src/state/lists/hooks.ts

```diff
@@ -60,9 +60,10 @@ function listToTokenMap(list: TokenList): TokenAddressMap {
     if (tokenMap[tokenInfo.chainId][formattedAddress] !== undefined) {
       return tokenMap
     }
-
-    const token = new WrappedTokenInfo(tokenInfo, list)
-    tokenMap[tokenInfo.chainId][formattedAddress] = token
+    try {
+      const token = new WrappedTokenInfo(tokenInfo, list)
```

Comment:
same here. tokenMap is guarantee that address is checksum. dont need try catch here

Reply:
This is main place where crash happended. The above `src/hooks/Tokens.ts` is just an additional.